types/ws/index.d.ts

```diff
@@ -64,17 +64,17 @@ declare class WebSocket extends events.EventEmitter {
     // Events
     on(event: 'error', cb: (err: Error) => void): this;
     on(event: 'close', cb: (code: number, message: string) => void): this;
-    on(event: 'message', cb: (data: any, flags: { binary: boolean }) => void): this;
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/16775 should have addressed these changes :)